jenkins/bootstrap.py

```diff
@@ -987,6 +980,10 @@ def parse_args(arguments=None):
         action='store_true',
         help='Clean the git repo before running tests.')
     args = parser.parse_args(arguments)
+    # --pull is deprecated, use --repo=k8s.io/foo=master:abcd,12:ef12,45:ff65
+    setattr(args, 'pull', None)
+    # --branch is deprecated, use --repo=k8s.io/foo=master
+    setattr(args, 'branch', None)
```

Comment:
per https://github.com/kubernetes/test-infra/pull/4482#issuecomment-328648313
This is a bit gross, but it should suffice to prevent usage while allowing a lot of otherwise valid but somewhat hairy tests to work until the port is in place.